cli/lib/kontena/cli/plugins/uninstall_command.rb

```diff
@@ -6,9 +6,11 @@ class UninstallCommand < Kontena::Command
     include Kontena::Cli::Common
 
     parameter 'NAME', 'Plugin name'
+
     option "--force", :flag, "Force remove", default: false, attribute_name: :forced
 
     def execute
+      exit_with_error "Plugin #{pastel.cyan(name)} is not installed" unless Kontena::PluginManager.instance.installed(name) && !ENV['NO_PLUGINS']
```

Comment:
BTW as a separate issue, it wouldn't hurt to drop the `Kontena::PluginManager#uninstall_plugin` and make this exit overridable with `--force`, because a broken plugin with `LoadError`'s won't be `installed`, and thus impossible to `kontena plugin uninstall` it ATM?